nav2_planner/src/planner_server.cpp

```diff
@@ -481,12 +481,73 @@ PlannerServer::computePlan()
+      ex.what());
+    result->error_code = nav2_msgs::action::ComputePathToPose::Goal::START_OCCUPIED;
+    action_server_pose_->terminate_current(result);
+  } catch (nav2_core::GoalOccupied & ex) {
+    RCLCPP_WARN(
+      get_logger(), "%s plugin failed to plan path. Goal Pose (%.2f, %.2f) was occupied: \"%s\"",
+      goal->planner_id.c_str(),
+      goal->goal.pose.position.x,
+      goal->goal.pose.position.y,
+      ex.what());
+    result->error_code = nav2_msgs::action::ComputePathToPose::Goal::GOAL_OCCUPIED;
+    action_server_pose_->terminate_current(result);
+  } catch (nav2_core::NoValidPathCouldBeFound & ex) {
+    RCLCPP_WARN(
+      get_logger(), "%s plugin failed to plan path. No Valid Path found: \"%s\"",
+      goal->planner_id.c_str(),
+      ex.what());
+    result->error_code = nav2_msgs::action::ComputePathToPose::Goal::NO_VALID_PATH;
+    action_server_pose_->terminate_current(result);
+  } catch (nav2_core::PlannerTimedOut & ex) {
+    RCLCPP_WARN(
+      get_logger(), "%s plugin failed to plan path. Timeout occurred: \"%s\"",
+      goal->planner_id.c_str(),
+      ex.what());
+    result->error_code = nav2_msgs::action::ComputePathToPose::Goal::TIMEOUT;
+    action_server_pose_->terminate_current(result);
+  } catch (nav2_core::StartOutsideMapBounds & ex) {
+    RCLCPP_WARN(
+      get_logger(), "%s plugin failed to plan path. Start Outside map: \"%s\"",
+      goal->planner_id.c_str(),
+      ex.what());
+    result->error_code = nav2_msgs::action::ComputePathToPose::Goal::START_OUTSIDE_MAP;
+    action_server_pose_->terminate_current(result);
+  } catch (nav2_core::GoalOutsideMapBounds & ex) {
+    RCLCPP_WARN(
+      get_logger(), "%s plugin failed to plan path. Goal Outside map: \"%s\"",
+      goal->planner_id.c_str(),
+      ex.what());
+    result->error_code = nav2_msgs::action::ComputePathToPose::Goal::GOAL_OUTSIDE_MAP;
+    action_server_pose_->terminate_current(result);
+  } catch (nav2_core::StartIsEqualToGoal & ex) {
+    RCLCPP_WARN(
+      get_logger(), "%s plugin failed to plan path. Start equal to goal: \"%s\"",
+      goal->planner_id.c_str(),
+      ex.what());
+    result->error_code = nav2_msgs::action::ComputePathToPose::Goal::START_IS_EQUAL_TO_GOAL;
+    action_server_pose_->terminate_current(result);
+  } catch (nav2_core::PlannerTFError & ex) {
+    RCLCPP_WARN(
+      get_logger(), "%s plugin failed to plan path. Start equal to goal: \"%s\"",
+      goal->planner_id.c_str(),
+      ex.what());
+    result->error_code = nav2_msgs::action::ComputePathToPose::Goal::TF_ERROR;
+    action_server_pose_->terminate_current(result);
   } catch (std::exception & ex) {
     RCLCPP_WARN(
-      get_logger(), "%s plugin failed to plan calculation to (%.2f, %.2f): \"%s\"",
-      goal->planner_id.c_str(), goal->goal.pose.position.x,
-      goal->goal.pose.position.y, ex.what());
-    action_server_pose_->terminate_current();
+      get_logger(), "%s plugin failed to plan: \"%s\"",
```

Comment:
Still add the goal pose here, it helps understand which request specifically failed